synapse/models/inet.py

```diff
@@ -1223,6 +1224,38 @@ def getModelDefs(self):
                     }),
                 ),
 
+                'interfaces': (
+
+                    ('inet:proto:request', {
+
+                        'doc': 'Properties common to network protocol requests and responses.',
+                        'interfaces': ('it:host:activity',),
+
+                        'props': (
+                            ('flow', ('inet:flow', {}), {
+                                'doc': 'The raw inet:flow containing the request.'}),
+                            ('client', ('inet:client', {}), {
+                                'doc': 'The inet:addr of the client.'}),
+                            ('client:ipv4', ('inet:ipv4', {}), {
+                                'doc': 'The server IPv4 address that the request was sent from.'}),
+                            ('client:ipv6', ('inet:ipv6', {}), {
+                                'doc': 'The server IPv6 address that the request was sent from.'}),
```

Comment:
Missing (optional) client:port ?

Reply:
Left out on purpose, we can discuss adding in a subsequent PR if it comes up.